go/service/gregor.go

```diff
@@ -521,21 +525,37 @@ func (g *gregorHandler) iterateOverFirehoseHandlers(f func(h libkb.GregorFirehos
 	g.firehoseHandlers = freshHandlers
 }
 
+func (g *gregorHandler) pushStateThread() {
+	var lastPush func()
+	var lastTime time.Time
+	for push := range g.pushStateCh {
+		lastPush = push
+		if time.Since(lastTime) > time.Second {
```

Comment:
if this is false we just lose the update, seems wrong

Reply:
I believe this is handled in my latest version